src/ansys/aedt/core/modeler/pcb/object_3d_layout.py

```diff
@@ -784,6 +833,15 @@ def set_solderball(
             ]
         return self.change_property(args)
 
+    def _get_model_info(self):
+        props = self._oeditor.GetComponentInfo(self.name)
+        model_info = ""
+        for p in props:
+            if "PortProp(" in p:
+                model_info = p
+                break
+        return model_info
+
```

Comment:
I like that you extracted lines of codes into sub methods ! Could you make this method private ?
Could you make this method a static methods of the class though ? I don't think they will be used in a different context than this class. Also, could you make them private if possible ?

Reply:
I can make this private but I don't think it can be static as it does depend the instance's `self.name` property.

Reply:
Yeah my bad, I did the comment while writting the following one (where the methods can be made static).
Just make it private :)

Reply:
Done.